sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/Transforms.java

```diff
@@ -275,19 +275,19 @@ private Transforms() {
      * @param textLine The service side text line object.
      * @return the custom type Appearance model.
      */
-    private static Appearance getAppearance(TextLine textLine) {
-        Style style = new Style();
+    private static TextAppearance getTextAppearance(TextLine textLine) {
+        TextStyle textStyle = new TextStyle();
         if (textLine.getAppearance() != null && textLine.getAppearance().getStyle() != null) {
             if (textLine.getAppearance().getStyle().getName() != null) {
-                StyleHelper.setName(style,
-                    TextStyle.fromString(textLine.getAppearance().getStyle().getName().toString()));
+                TextStyleHelper.setName(textStyle,
+                    TextStyleName.fromString(textLine.getAppearance().getStyle().getName().toString()));
             }
-            StyleHelper.setConfidence(style, textLine.getAppearance().getStyle().getConfidence());
+            TextStyleHelper.setConfidence(textStyle, textLine.getAppearance().getStyle().getConfidence());
         }
 
-        Appearance appearance = new Appearance();
-        AppearanceHelper.setStyle(appearance, style);
-        return appearance;
+        TextAppearance textAppearance = new TextAppearance();
+        TextAppearanceHelper.setStyle(textAppearance, textStyle);
+        return textAppearance;
```

Comment:
if `textLine.getAppearance() == null` what will be the default value for the object `textAppearance` ?

Reply:
It would be null, do we set any other default value?

Reply:
I see. Then it might be me not knowing Java. if I do this in .NET `TextAppearance textAppearance = new TextAppearance();` it will create the type with default values.
I will have to do 
```
if (textLine.getAppearance() == null)
  return null;
```
explicitly for it to work.
if this is not the case for Java, ignore :)